### DISCUSSION TITLE
ttkbootstrap convert to .exe with aotu-py-to-exe

### DISCUSSION THREAD

**Discussion by Priyaskd108** [israel-dryer/ttkbootstrap#507] [Q&A]
Hi.....I'm having issues after converting to exe with auto-py-to-exe. When it's run on different computers with Windows operating system, the buttons and Text are displaced or appear larger.

**Comment by TheDemonGuard** [comment]
> Hi.....I'm having issues after converting to exe with auto-py-to-exe. When it's run on different computers with Windows operating system, the buttons and Text are displaced or appear larger.

Is it possible to try with the latest version?
If the issue still persists, please provide steps/instructions to reproduce the issue along with a demo/sample script.
